ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -62,6 +66,7 @@ void RSkComponentImage::OnPaint(
       canvas->drawImageRect(imageData,targetRect,&paint);
       canvas->restore();
     } else {
+      imageEventEmitter->onError(); /*Emitting Image Load failed Event*/
```

Comment:
Is there an error event required for the below condition at line 39 instead of assert 
"assert(source.uri.substr(0, 14) == "file://assets/");"